src/testing/utils.tsx

```diff
@@ -76,6 +79,8 @@ export const wrapComponent = (
     options && "store" in options
       ? options.store
       : createStore(options?.state ?? rootStateFactory.build());
+  const config = store.getState().general.config || configFactory.build();
+  initialiseAuthFromConfig(config as WindowConfig, store.dispatch);
```

Comment:
`WindowConfig` is defined as `type WindowConfig = Omit<Config, "authMethod">` so once we remove authMethod from config in [WD-20710](https://warthogs.atlassian.net/browse/WD-20710) we can also remove `as WindowConfig` so maybe add a TODO to remember to do that?